### PR TITLE
[TIMOB-24738] iOS: Expose popToRootWindow in Ti.UI.Tab

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24738

### CODE REVIEW

File: apidoc/Titanium/UI/Tab.yml [hunk 1]
```diff
@@ -126,6 +126,17 @@ methods:
...
         type: Titanium.UI.Window
 
+  - name: popToRootWindow
+    summary: Closes all windows that are currently opened inside the tab.
+    description: Note that only the `close` event of the most recently opened window is fired.
+    parameters:
+      - name: options
+        summary: |
+            Options supporting a single `animated` boolean property to determine whether the windows
+            will be animated (default) while being closed (default: false).
```
Comment (vijaysingh-axway) [inline apidoc/Titanium/UI/Tab.yml]: I think word "(default)" is not required in above sentence.

**Review by vijaysingh-axway — APPROVED** [review]
CR and FT passed.

File: apidoc/Titanium/UI/Tab.yml (hunk 1 above)
Comment (hansemannn) [inline apidoc/Titanium/UI/Tab.yml]: You are right :-D

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Review by htbryant94 — APPROVED** [review]
Approved! Ti.UI.Tab.popToRootWindow now allows for closing all windows at once. Additionally tested with 50 windows open simultaneously, and was able to close all windows without error.

**Pull request merged**